client/src/plugins/error-tracking/__tests__/ErrorTrackingSpec.js

```diff
@@ -357,8 +360,85 @@ describe('<ErrorTracking>', () => {
       { key: 'plugins', value: plugins.map(({ name }) => name) }
     );
   });
+
+
+  describe('Path normalization', () => {
+
+    it('should normalize Windows paths', () => {
+
+      return normalizationTest('/C:/Users/user/test-user/Desktop/Camunda/resources/app.asar/public/');
```

Comment:
Just wondering, can this fail due to timeouts? Since `normalizationTest` is asynchronous, but the test is not.

Reply:
AFAIK Mocha waits for the test to finish if you return a promise. Since normalizaionTest is async, it should return a Promise. So if this method throws an exception the test also fails.

Reply:
> AFAIK Mocha waits for the test to finish if you return a promise. 

Man, I always forget this aspect.

Reply:
Windows paths look a bit different.

```suggestion
      return normalizationTest('C:\\Users\user\\test-user\\Desktop\\Camunda\\resources\\app.asar\\public');
```

Reply:
I copied this path from actual Sentry errors but maybe Sentry prints it like this. I'll add this as an additional case rather than converting the case.

Reply:
Sure. It seems they are Unix-centric ;)

Reply:
Done via https://github.com/camunda/camunda-modeler/pull/1846/commits/44d5a4c5a41f2c8a6c3ea68b62582c12361eee66